src/sdkUnitTests/sdkUnitTests.sharpmake.cs

```diff
@@ -0,0 +1,41 @@
+using Sharpmake;
+
+[module: Include("../sdk/sdk.sharpmake.cs")]
+
+namespace MnemosyneSdk
+{
+	[Generate]
+	public class SdkUnitTests : Project
```

Comment:
What about introducing base class for both lib and PE and set common properties there?

Reply:
done